articles/becoming-a-flutter-developer/index.md

```diff
@@ -0,0 +1,188 @@
+
+- [Bitbucket](https://bitbucket.org/)
+
+2. **Flutter Framework**
+
+Flutter is a UI framework used to develop high performance and high-quality native applications for Android, iOS, web and desktop using a single code base. Dart is the primary programming language for Flutter Framework. 
+
+Some features of Flutter that make it popular include 
+
+1. `Hot Reloading` which enables developers to load code changes quickley and preserves the state of the app. 
+2. `Hot Restart`. This is similar to hot reloading but causes the app to lose its state and takes more time.
+3. `Present UI element` to create beautiful app designs for better `UI and UX`. 
+
+These UI designs are 
+1. Cupertino 
+2. Material Design. 
+
+These built-in features make building Flutter applications fast in performance. 
+
+
+3. **Integrated Development Environment**
+
+In using Flutter, you will choose between three IDEs. Which are:
+- Android Studio.
+- Visual Studio Code.
+- IntelliJ IDEA.
+
+Most Flutter developers pick either Android Studio and Visual Studio Code. They chose IDEs depending on the features suitable to them. Android Studio and Visual Studio Code are both capable of creating full Flutter apps, developer preferences aside.
+
+4. **Flutter DevTools**
+
+It is a tooling web suite for Flutter and Dart developers comprising a layout inspection tools, performance tools and memory tools. It contains all the debugging tools that you need to be an efficient and effective Flutter developer.
+
+Some of tools in the [Flutter DevTools web suite](https://flutter.dev/docs/development/tools/devtools/overview) include:
+
+1. Flutter Inspector - It is used to diagnose and debug layout issues.
+
+2. Timeline - It diagnoses performance problems which are the most common kinds during development. The Timeline shows the amount of time taken by each widget.
+
+
+5. **Design Tools**
+
+These are tools used to craft and visualize ideas. Design is an important stage of app development. It ensures for a smooth software development process. Some software development teams include UI/UX designers, especially for building commercial applications while others lack. Building personal projects will also need knowledge of design tools.
+
+Here are some of the most popular design tools:
+- [Sketch.](https://www.sketch.com/) This is however only available for Mac.
+- [Adobe XD.](https://www.sketch.com/)
+- [Figma.](https://www.figma.com/) 
+- [Flutter Studio.](https://flutterstudio.app/)
+
+After designing, you can use [Supernova](https://www.supernova.io/) to convert the design into code by exporting it. You will, however, need to refactor the exported code to break it into readable code. Supernova is compatible with the design tools mentioned above. 
```

Comment:
You can remove this part entirely.
`After designing, you can use [Supernova](https://www.supernova.io/) to convert the design into code by exporting it. You will, however, need to refactor the exported code to break it into readable code. Supernova is compatible with the design tools mentioned above. ` 
It is not much involved in `what it takes to be a Flutter developer`.